pydantic/networks.py

```diff
@@ -352,6 +353,11 @@ class PostgresDsn(AnyUrl):
     user_required = True
 
 
+class RabbitMqDsn(AnyUrl):
+    allowed_schemes = {'amqp'}
+    user_required = True
```

Comment:
According to @nuno-andre username is not required, personally I have no idea.

Reply:
The AMQP spec: https://www.rabbitmq.com/uri-spec.html

It's not required. I've removed.

Reply:
Well, I've reread:
> The host to which the underlying TCP connection is made is determined from the host component according to RFC3986, section 3.2.2. Note that according to the ABNF, the host component may not be absent, but it may be zero-length.

So for us, we don't need the `host_required` I guess.

Reply:
`allowed_schemes` should be `{'ampq', 'ampqs'}`. From specs (Section 4):

> _The "amqps" URI scheme is used to instruct a client to make an secured connection to the server._